createdisk.sh

```
@@ -49,6 +49,18 @@ VM_IP=$(sudo virsh domifaddr ${VM_NAME} | tail -2 | head -1 | awk '{print $4}' |
 
 wait_for_ssh ${VM_NAME} ${VM_IP}
 
+# Configure proxy when at least one URL is set
+if [[ -n "${SNC_HTTP_PROXY}" || -n "${SNC_HTTPS_PROXY}" ]]; then
+    ${SSH} core@${VM_IP} 'sudo bash -x -s' <<EOF
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)